api/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java

```diff
@@ -632,6 +632,25 @@ public final class NessieConfigConstants {
   @ConfigItem
   public static final String CONF_NESSIE_CLIENT_API_VERSION = "nessie.client-api-version";
 
+  /**
+   * Override all commit messages with the configured value. The corresponding HTTP header is {@code
+   * Nessie-Commit-Message}.
+   */
+  @ConfigItem public static final String CONF_NESSIE_COMMIT_MESSAGE = "nessie.commit-meta.message";
+
+  /**
+   * Set/override the author of all commits. The corresponding HTTP header is {@code
+   * Nessie-Commit-Authors}.
+   */
+  @ConfigItem public static final String CONF_NESSIE_COMMIT_AUTHOR = "nessie.commit-meta.author";
```

Comment:
nit: `authors` (with an s)?.. Config properties are not repeatable (unlike headers)